pkg/network/notary_request_pool.go

```diff
@@ -0,0 +1,244 @@
+package network
```

Comment:
I remember this pool in a separate package, why do we need it in the `network`? It can be `network/notarypool` or something like that.

Reply:
This was mostly done for testing. Now `NotaryPool` is in `network/notarypool`, but we need a Blockchainer for primitive tests. At the same time we cannot move `notarypool_test.go` to the `network` package because we need access to private fields of the pool. As a result, I had to copy `network.testchain` to the `notarypool` package.

Reply:
Move it out to `internal` then, we have a number of testing packages there shared among several real packages and some `fakechain` is appropriate there.